sycl/examples/simple-dpcpp-app.cpp

```diff
@@ -0,0 +1,54 @@
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+int main() {
+  // Create a buffer of 4 ints to be used inside the kernel code.
+  buffer<cl_int, 1> Buffer(4);
+
+  // Create a simple asynchronous exception handler.
+  auto AsyncHandler = [](exception_list ExceptionList) {
+    for (auto &Exception : ExceptionList) {
+      std::rethrow_exception(Exception);
```

Comment:
So, the very first exception will thrown only, isn't it?

Reply:
@s-kanaev I'm not sure there's a way to re-throw all exceptions in the list.

Reply:
I'm OK with that but it should be noted that the user might want to handle exceptions somehow. Anyway, I'm not pushing onto it.

Reply:
This sample only shows basic API usage. I'm not sure it is a suitable place to educate users about variety of possibilities provided by C++.